addons/website_forum/tests/test_controller.py

```diff
@@ -47,6 +47,6 @@ def test_02_admin_attachment(self):
                                                         'mimetype': 'text/plain', 'res_model': 'forum.post',
                                                         'raw': self.pixel, 'website_id': 1}])
         domain = [('name', '=', 'test_pixel')]
-        result = attachment.search_read(domain)
+        result = attachment.search(domain)
         self.assertTrue(len(result), "No attachment fetched")
-        self.assertEqual(result[0]['id'], attachment.id)
+        self.assertEqual(result[0].id, attachment.id)
```

Comment:
If above diff is applied
```suggestion
        self.assertEqual(result.id, attachment.id)
```